README.md

```diff
@@ -0,0 +1,162 @@
+<a name="readme-top"></a>
+
+<div align="center">
+  <h3><b>Budget App</b></h3>
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+    - [Live Version](#live-version)
+- [💻 Getting Started](#getting-started)
+  - [Prerequisites](#prerequisites)
+  - [Setup](#setup)
+  - [Install](#install)
+  - [Usage](#usage)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 Budget App <a name="about-project"></a>
+
+**Budget App** is an web app built with rails and uses postgresql. The user can log in, create new categories and new records for their expenses. Each category has a name and an icon, and each expense has a name, amount and a category. The user also can see the amount for each category and the total amount for all categories.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://ruby.org/">Ruby</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+- **The page is built with Ruby on Rails**
+- **The desing is for mobile first approach**
+- **For handling user's log and register the App uses device gem**
+
+### Live demo <a name="live-demo">
+[Budget App](https://budgetapp-txbb.onrender.com/)
+</a>
+
+### Presentation Video <a name="presentation">
+[Budget App](https://www.loom.com/share/c8fc171a9e044664a272c7fb4677a085)
+</a>
```

Comment:
- [ ] Please make sure to highlight some interesting piece of code or something you built that you are very proud of as requested by the previous reviewer and as per this [requirement](https://github.com/microverseinc/curriculum-rails/blob/main/capstone/rails_capstone.md#:~:text=you%20should%20also%20highlight%20some%20interesting%20piece%20of%20code%20or%20something%20you%20built%20that%20you%20are%20very%20proud%20of.).